src/main/java/org/opentripplanner/standalone/config/framework/json/NodeInfoBuilder.java

```diff
@@ -63,6 +65,21 @@ NodeInfoBuilder withDescription(String description) {
     return this;
   }
 
+  String description() {
+    if (!experimentalFeature) {
+      return description;
+    }
+    if (description == null) {
+      return EXPERIMENTAL_FEATURE;
+    }
+    return description + "\n\n" + EXPERIMENTAL_FEATURE;
```

Comment:
Do you want to make this bold?

Reply:
Krank it to eleven.

Reply:
That is maybe a good idea. I vill change the text.